src/pages/home/HeaderView.js

```diff
@@ -52,20 +52,28 @@ const HeaderView = props => (
                 </TouchableOpacity>
             )}
             {props.report && props.report.reportName ? (
-                <View style={[
-                    styles.flex1,
-                    styles.flexRow,
-                    styles.alignItemsCenter,
-                    styles.justifyContentBetween,
-                ]}
+                <View
+                    style={[
+                        styles.flex1,
+                        styles.flexRow,
+                        styles.alignItemsCenter,
+                        styles.justifyContentBetween,
+                    ]}
                 >
-                    <View style={[styles.flex1]}>
-                        <Text numberOfLines={1} style={[styles.navText]}>
+                    <View
+                        style={[styles.flex1]}
+                    >
+                        <Text
+                            numberOfLines={2}
+                            style={[styles.navText]}
+                        >
                             {props.report.reportName}
                         </Text>
                     </View>
 
-                    <View style={[styles.reportOptions, styles.flexRow]}>
+                    <View
+                        style={[styles.reportOptions, styles.flexRow]}
```

Comment:
If you're going to put one style per line let's be consistent about it here.

Reply:
Hi @deetergp 

The formatting inconsistencies have been fixed.

Thanks for your feedback